ui: landing: debug panel

gioelecerati opened the pull request:
- Adding admin debug panel for daydream

Open issues:
- The payload is static, would be good to have that JSON editable and a button "Apply" to send its parameters to the control api. Maybe even display the parameter in a separate editable container for clarity

mjh1 commented:
@gioelecerati is the `str_` prefix being removed from the stream ID?
<img width="280" alt="Screenshot 2025-02-25 at 11 51 09" src="https://github.com/user-attachments/assets/8cfe2e3d-157f-4225-b6a9-329ca6237371" />

While you're here would you be able to add the stream key to this debug pane too? thanks!

gioelecerati commented:
> @gioelecerati is the `str_` prefix being removed from the stream ID? <img alt="Screenshot 2025-02-25 at 11 51 09" width="280" src="https://private-user-images.githubusercontent.com/3630965/416666257-8cfe2e3d-157f-4225-b6a9-329ca6237371.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDA0ODQ2MzksIm5iZiI6MTc0MDQ4NDMzOSwicGF0aCI6Ii8zNjMwOTY1LzQxNjY2NjI1Ny04Y2ZlMmUzZC0xNTdmLTQyMjUtYjZhOS0zMjljYTYyMzczNzEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDIyNSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTAyMjVUMTE1MjE5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9M2RmNmUyNDcyOTJjMWVlNDRmNjc2ODJlNjA4Y2Y1NTcwN2M1M2U3OWNmNmYzMGI5OTg5MmJhMWI0YzdiZTE3ZSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.8KhpfnufcVsKx5ficCM17tYm2yZjL-0oMEqdc3xPB10">
> 
> Also while you're here would you be able to add the stream key to this debug pane too? thanks!

Ah good catch! I was passing the playbackId instead of the streamID
Fixed it and added the streamkey
